OpenOversight/app/main/forms.py

```diff
@@ -514,6 +514,27 @@ class OOIdForm(Form):
 
 
 class IncidentForm(DateFieldForm):
+    # In line validator for the officers field. Check to see if the officers are within the selected department
+    def validate_officers(form, field):
```

Comment:
Add a return type to the `validate_officers` function.

Reply:
Please let me know if the additional changes are satisfactory! Thanks!

Reply:
It looks good! It looks like the tests are failing and it'd be ideal if you could use the `faker` package on the tests, similar to what you'll see on the others.